Automattic/wp-calypso
# LPM API Migration: Replace the Source API version of Bancontact

Related to # https://github.com/Automattic/payments-shilling/issues/2976

## Proposed Changes

* In order to use Bancontact with the Payments Intent API, the Stripe Elements for iDEAL in the payment selector form needs to be directed to use the class `WPCOM_Billing_Stripe_Bancontact` when processing the transaction


## Why are these changes being made?


* Stripe is retiring their Sources API in favor of the PaymentIntent API.


## Testing Instructions


- Apply D159715-code to your sandbox
- Enable Store Sandbox
- Start up local Calypso with this PR applied
- Add a product to your cart and proceed to Checkout
- Use the country `Belgium` and the postal code `9034`
- In the Payment Options section, select iDEAL and fill in your name and a bank
- You should be redirected to an Authorization page. It should mention `payment_intent` on it

![PR93967](https://github.com/user-attachments/assets/c2e42c8f-a898-4822-9aa8-d69fe345db92)


## Pre-merge Checklist
